contrib/kind.sh

```diff
@@ -1064,28 +1080,44 @@ install_metallb() {
     kubectl label node "$n" node.kubernetes.io/exclude-from-external-load-balancers-
   done
 
-  local frr_ips
-  frr_ips=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}#{{end}}' frr)
-  echo "${frr_ips}"
-  local kind_network
-  kind_network=$(echo "${frr_ips}" | cut -d '#' -f 2)
-  echo "${kind_network}"
-  local client_network
-  client_network=$(echo "${frr_ips}" | cut -d '#' -f 1)
-  echo "${client_network}"
-  # The following only works for single stack, to be modified if dualstack is to be supported:
-  local client_subnet
-  client_subnet=$(docker network inspect clientnet -f '{{range .IPAM.Config}}{{.Subnet}}{{end}}')
-  echo "${client_subnet}"
+  kind_network_v4=$(docker inspect -f '{{index .NetworkSettings.Networks "kind" "IPAddress"}}' frr)
+  echo "FRR kind network IPv4: ${kind_network_v4}"
+  kind_network_v6=$(docker inspect -f '{{index .NetworkSettings.Networks "kind" "GlobalIPv6Address"}}' frr)
+  echo "FRR kind network IPv6: ${kind_network_v6}"
+  local client_network_v4 client_network_v6
+  client_network_v4=$(docker inspect -f '{{index .NetworkSettings.Networks "clientnet" "IPAddress"}}' frr)
+  echo "FRR client network IPv4: ${client_network_v4}"
+  client_network_v6=$(docker inspect -f '{{index .NetworkSettings.Networks "clientnet" "GlobalIPv6Address"}}' frr)
```

Comment:
@kyrtapz : Is this interim error expected and ok:?
```
Status: Downloaded newer image for quay.io/frrouting/frr:9.0.2
for frr in $(docker ps -a -f name=frr --format {{.Names}}) ; do     docker rm -f $frr ; done
docker run -d --privileged --network kind --rm --ulimit core=-1 --name frr --volume /tmp/tmp.Bkxk1K05En/frr-volume:/etc/frr quay.io/frrouting/frr:9.0.2
505d34ffa6af7ba669c28dc9c5aa594dc0819a5c41a5cf447629998c010eeed9
docker inspect -f "{{ range .NetworkSettings.Networks}}{{.GlobalIPv6Address}}{{end}}" frr
fc00:f853:ccd:e793::5
Error from server (InternalError): error when creating "/tmp/metallb/jo2MP9/metallb/dev-env/bgp/config.yaml": Internal error occurred: failed calling webhook "bgppeersvalidationwebhook.metallb.io": failed to call webhook: Post "[https://metallb-webhook-service.metallb-system.svc:443/validate-metallb-io-v1beta2-bgppeer?timeout=10s](https://metallb-webhook-service.metallb-system.svc/validate-metallb-io-v1beta2-bgppeer?timeout=10s)": context deadline exceeded
bgpadvertisement.metallb.io/empty created
Error from server (InternalError): error when creating "/tmp/metallb/jo2MP9/metallb/dev-env/bgp/config.yaml": Internal error occurred: failed calling webhook "ipaddresspoolvalidationwebhook.metallb.io": failed to call webhook: Post "[https://metallb-webhook-service.metallb-system.svc:443/validate-metallb-io-v1beta1-ipaddresspool?timeout=10s](https://metallb-webhook-service.metallb-system.svc/validate-metallb-io-v1beta1-ipaddresspool?timeout=10s)": context deadline exceeded
Sleeping for 2s
bgppeer.metallb.io/peer created
ipaddresspool.metallb.io/dev-env-bgp created
bgpadvertisement.metallb.io/empty unchanged
+ docker network create --subnet=172.22.0.0/16 --ipv6 --subnet=fc00:f853:ccd:e792::/64 --driver bridge clientnet
fb3b3e2f56c2593f8b737f29a0944c600e9a8880d7387dabcf15b23a6535870f
+ docker network connect clientnet frr
+ docker run --cap-add NET_ADMIN --user 0 -d --network clientnet --rm --name lbclient quay.io/itssurya/dev-images:metallb-lbservice
Unable to find image 'quay.io/itssurya/dev-images:metallb-lbservice' locally
metallb-lbservice: Pulling from itssurya/dev-images
```

Reply:
nit: we could avoid trying to get the v6 information on the v4 lanes but I am ok if you say "let's let it be" :)
```
+ local client_network_v4 client_network_v6
++ docker inspect -f '{{index .NetworkSettings.Networks "clientnet" "IPAddress"}}' frr
+ client_network_v4=172.22.0.2
FRR client network IPv4: 172.22.0.2
+ echo 'FRR client network IPv4: 172.22.0.2'
++ docker inspect -f '{{index .NetworkSettings.Networks "clientnet" "GlobalIPv6Address"}}' frr
+ client_network_v6=
FRR client network IPv6: 
+ echo 'FRR client network IPv6: '
+ local client_subnets
++ docker network inspect clientnet -f '{{range .IPAM.Config}}{{.Subnet}}#{{end}}'
+ client_subnets=172.22.0.0/16#
172.22.0.0/16#
+ echo 172.22.0.0/16#
+ local client_subnets_v4 client_subnets_v6
++ echo 172.22.0.0/16#
++ cut -d '#' -f 1
+ client_subnets_v4=172.22.0.0/16
+ echo 'client subnet IPv4: 172.22.0.0/16'
client subnet IPv4: 172.22.0.0/16
++ echo 172.22.0.0/16#
++ cut -d '#' -f 2
+ client_subnets_v6=
+ echo 'client subnet IPv6: '
client subnet IPv6: 
++ kind get nodes --name ovn
```